app/overlay-zephyr-modem-nrf9160dk-nrf52840.conf

```diff
@@ -4,5 +4,28 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-# nRF52 <=> nRF91 interface pin 4 (see https://docs.nordicsemi.com/bundle/ug_nrf91_dk/page/UG/nrf91_DK/board_controller.html)
-CONFIG_SM_POWER_PIN=22
+CONFIG_USE_SEGGER_RTT=n
+# Where console messages (printk) are output.
+# By itself, SLM does not output any.
+CONFIG_RTT_CONSOLE=n
+CONFIG_UART_CONSOLE=y
+# Where SLM logs are output.
+CONFIG_LOG_BACKEND_RTT=n
+CONFIG_LOG_BACKEND_UART=y
+
+CONFIG_SHELL=y
+CONFIG_GPIO=y
+CONFIG_GPIO_SHELL=y
+CONFIG_PM_DEVICE_SHELL=y
+
+CONFIG_UART_1_INTERRUPT_DRIVEN=n
+CONFIG_UART_1_ASYNC=y
+CONFIG_UART_USE_RUNTIME_CONFIGURE=y
+CONFIG_UART_ASYNC_API=y
+
+CONFIG_SM_SMS=n
+CONFIG_SM_GNSS=n
+CONFIG_SM_NRF_CLOUD=n
+CONFIG_SM_GPIO=n
+
+CONFIG_SIZE_OPTIMIZATIONS_AGGRESSIVE=y
```

Comment:
Remove this whole file.

removed lines are not needed anymore and added lines are unrelated to this PR.